zet

```diff
@@ -212,7 +212,8 @@ x_push() {
 x_read() {
     readarray -t directories <<< "$(x_all)"
     if [[ " ${directories[*]} " =~ " $1 " ]]; then
-        $ZET_EDITOR -R "$(x_dir)/$1/README.md"
+        # $ZET_EDITOR -R "$(x_dir)/$1/README.md"
+	xdg-open $GITHUB/tree/main/$1 2> /dev/null
```

Comment:
Zet user may not have $GITHUB, can we keep $(x_dir). Why change this aspect?

Reply:
I can rename the variables to apply more broadly. It doesn't have to be Github. It's meant to be the URL to whatever open-source hosting service they're using.

I made this change for quick access to view any zettel on github, where it's easier and more pleasant to navigate. Besides, on the main branch, there isn't much utility in x_read when we already have x_edit; only write-protection.